src/store/ingresses.js

```diff
@@ -30,8 +31,14 @@ const ingresses_container = {
             let data = {}
             data['name'] = ingress.name
             data['tls'] = ingress.tls
+            data['creation_timestamp'] = moment(ingress.creation_timestamp).format()
             data['annotations'] = ingress.annotations.map(anno => { return `${anno['key']}: ${anno['value']}` })
-            data['hosts'] = ingress.rules.map(rule => { return rule.host })
+            data['rules'] = ingress.rules
+            data['hosts'] = ingress.rules.map(rule => {
+              return data['tls'] === true
+                ? `<a href=https://${rule.host}>https://${rule.host}</a>`
+                : `<a href=http://${rule.host}>https://${rule.host}</a>`
```

Comment:
I think you forgot to remove the 's' if tls is false, like this:
`<a href=http://${rule.host}>http://${rule.host}</a>`

I also think you can change `data['tls'] === true` to `data['tls']` since it will be either true or false